src/service/configInjection.ts

```diff
@@ -74,6 +74,12 @@ export async function setup(trx: Knex.Transaction, context: SynchronizeContext):
 
   const values = convertToKeyValuePairs(injectedConfig.vars);
 
+  if (!values.length) {
+    log(`Config not available. Skipping insertion on ${INJECTED_CONFIG_TABLE} table.`);
+
+    return;
```

Comment:
Let's exit here with `process.exit(1)` because the setup function's return type is void.

Reply:
Won't this abruptly end the process? We need to just skip this step and let the task continue.

Reply:
Ah good point. Let's keep it as it is.